### DISCUSSION TITLE
temporal is keeping printing Unspecified task queue msg

### DISCUSSION THREAD

**Discussion by sivankumar86** [airbytehq/airbyte#38587] [Connector Questions]
I have upgraded Airbyte to 0.60.x and temporal logs keep printing below msg. Could you shed some light on this ? 

{"level":"warn","ts":"2024-05-22T22:49:08.384Z","msg":"Unspecified task queue kind","service":"frontend","wf-task-queue-name":"CHECK_CONNECTION","wf-namespace":"default","logging-call-at":"workflow_handler.go:3772"}
{"level":"warn","ts":"2024-05-22T22:49:09.281Z","msg":"Unspecified task queue kind","service":"frontend","wf-task-queue-name":"CONNECTION_UPDATER","wf-namespace":"default","logging-call-at":"workflow_handler.go:3772"}
{"level":"warn","ts":"2024-05-22T22:49:09.652Z","msg":"Unspecified task queue kind","service":"frontend","wf-task-queue-name":"DISCOVER_SCHEMA","wf-namespace":"default","logging-call-at":"workflow_handler.go:3772"}
{"level":"warn","ts":"2024-05-22T22:49:09.975Z","msg":"Unspecified task queue kind","service":"frontend","wf-task-queue-name":"NOTIFY","wf-namespace":"default","logging-call-at":"workflow_handler.go:3772"}
{"level":"warn","ts":"2024-05-22T22:49:10.552Z","msg":"Unspecified task queue kind","service":"frontend","wf-task-queue-name":"CONNECTION_UPDATER","wf-namespace":"default","logging-call-at":"workflow_handler.go:3772"}
{"level":"warn","ts":"2024-05-22T22:49:10.552Z","msg":"Unspecified task queue kind","service":"frontend","wf-task-queue-name":"CHECK_CONNECTION","wf-namespace":"default","logging-call-at":"workflow_handler.go:3772"}
{"level":"warn","ts":"2024-05-22T22:49:10.573Z","msg":"Unspecified task queue kind","service":"frontend","wf-task-queue-name":"DISCOVER_SCHEMA","wf-namespace":"default","logging-call-at":"workflow_handler.go:3772"}
{"level":"warn","ts":"2024-05-22T22:49:10.781Z","msg":"Unspecified task queue kind","service":"frontend","wf-task-queue-name":"GET_SPEC","wf-namespace":"default","logging-call-at":"workflow_handler.go:3772"}
{"level":"warn","ts":"2024-05-22T22:49:10.967Z","msg":"Unspecified task queue kind","service":"frontend","wf-task-queue-name":"NOTIFY","wf-namespace":"default","logging-call-at":"workflow_handler.go:3772"}

**Comment by oruen** [comment]
I believe it is related to this discussion https://github.com/temporalio/temporal/issues/5903
The cause is Java clients that don't set queue kind properly.

[a comment by an automated account is not shown]
